### DISCUSSION TITLE
can not change default value

### DISCUSSION THREAD

**Discussion by shahin203** [crater-invoice-inc/crater#1282] [General]
Does anybody know how to change the crater default currency value?
![Screenshot 2024-02-02 at 9 22 00 AM](https://github.com/crater-invoice/crater/assets/64118482/dea993a5-cfb7-46cd-9101-31bb94de12c9)

**Comment by gsluthra** [comment]
It seems in Code there is a check that says, if transactions are recorded, then disable the currency field: 
https://github.com/crater-invoice/crater/blob/master/app/Http/Controllers/V1/Admin/Settings/UpdateCompanySettingsController.php#L33

I guess, I am not sure how you can delete the transactions and then re-enable the field. Maybe someone from Crater core group can answer.

**Reply by shahin203** [reply]
"Please help me with my problem."
